src/VscProcess.cpp

```diff
@@ -209,13 +235,29 @@ bool VscProcess::KeyString(KeyString::Request& req, KeyString::Response& res)
   return true;
 }
 
+bool VscProcess::vscSettingsSrv(GetVscSettings::Request  &req, GetVscSettings::Response &res)
+{
+  res.srv_ready = srv_ready;
+  res.serial_number = serial;
+  res.firmware_version = firmware;
+  res.radio_power_level = radio_power_level;
+
+  return true;
+}
+
 void VscProcess::processOneLoop(const ros::TimerEvent&)
 {
   // Send heartbeat message to vehicle in every state
   vsc_send_heartbeat(vscInterface, myEStopState);
 
   // Check for new data from vehicle in every state
   readFromVehicle();
+
+  if (have_firmware && have_radio_power_level && have_firmware && !srv_ready)
+  {
+    srv_ready = true;
+    ROS_INFO("Settings Grabbed from VSC, service is ready..");
+  }
```

Comment:
Not sure on the rest of the member variables inside of the current repo, but in the mower repo we have been conforming to having member variables with underscores at the end.

`have_firmware_ && have_radio_power_level_ && have_firmware_`

Can you please update it to that since it makes it easier to read. Also the other ones added in the PR like 
`serial_` , `firmware_`, etc.

Update : 
Hmmm. noticing that the publishers and subscriber member variables are using the underscore convention. Might just need to hold off on it if there are too many member variables without that convention.

Reply:
Changing all of the previously existing formatting to conform with the `robotic-lawnmower` repo's formatting would probably be out of the scope of this PR, and we should probably have another PR for it, if necessary. I'd rather opt to keep the formatting uniform within the submodule for now.